03week/ticTacToe.js

```diff
@@ -23,24 +27,72 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
+//The function should take all outcomes from each row and decide if they are true
+// and should just need an if statement. 
+
 function horizontalWin() {
   // Your code here
+  if( board[0][0] == playerTurn && board[0][1] == playerTurn && board[0][2] == playerTurn ||
```

Comment:
you're repeating yourself a lot here, just return the evaluation. How else could you simplify?